Equilobe.DailyReport.Models/Jira/JiraBoard.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Equilobe.DailyReport.Models.Jira
+{
+    public class JiraBoard
+    {
+        public long maxResults { get; set; }
```

Comment:
@georgiantanase 
I think it is better to capitalize the property names. And please test if deserialization works that way